### DISCUSSION TITLE
Request: FIDO2 webauthn support

### DISCUSSION THREAD

**Discussion by Photograaf16** [goauthentik/authentik#3388] [Ideas]
Currently im still a authelia user but would love to switch over to authentik for the single-login feature on sonarr for example.
Only thing that is holding me back is because authentik doesnt support 2FA FIDO2 webauthn usb sticks.

I dont know if this is the correct spot to suggest this feature, but if so... again, i would switchover as soon as you support those USB's
Greetings from Holland!

**Comment by Cyberzoid1** [comment]
Hello, Passing by. Authentik supports WebAuthn which is a main component of FIDO2
https://goauthentik.io/docs/flow/stages/authenticator_webauthn/
